src/main/java/HelloWebDriver.java

```diff
@@ -0,0 +1,25 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+    public class HelloWebDriver {
+
+        public static void main(String[] args)  {
+
+            System.setProperty("webdriver.chrome.driver", "src\\test\\resources\\webdrivers\\chromedriver.exe");
+            WebDriver driver = new ChromeDriver();
+            driver.get("http://google.com");
```

Comment:
set https for url's, it can be potential reason of captcha appearing

Reply:
After more than 1 hour of refactoring the captcha is "disappeared", thank you!